modules/programs/lesspipe.nix

```diff
@@ -8,12 +8,25 @@ with lib;
   options = {
     programs.lesspipe = {
       enable = mkEnableOption "lesspipe preprocessor for less";
+
+      colorizer = mkOption {
+        type = types.str;
```

Comment:
Need `types.nullOr types.str` here otherwise `colorizer = null` is not a valid value.